packages/trader/src/Modules/Contract/Components/AccumulatorsStats/ticks-history-counter.jsx

```diff
@@ -2,27 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const TicksHistoryCounter = ({ has_progress_dots, value }) => (
-    <div
-        data-testid='dt_accu_stats_history_counter'
-        className={classNames('accumulators-stats__history-counter', {
-            'accumulators-stats__history-counter--emphasized': has_progress_dots && value === 0,
-        })}
-    >
-        {value}
-        {has_progress_dots && (
-            <div className='accumulators-stats__progress-dots'>
-                {[1, 2, 3].map(dot => {
-                    return <span key={dot} className={`dot-${dot}`} />;
-                })}
-            </div>
-        )}
-    </div>
-);
+const TicksHistoryCounter = ({ has_progress_dots, value, should_emphasize_last_counter }) => {
+    const should_highlight_last_counter = should_emphasize_last_counter && has_progress_dots && value === 0;
+    return (
+        <div
+            data-testid='dt_accu_stats_history_counter'
+            className={classNames('accumulators-stats__history-counter', {
+                'accumulators-stats__history-counter--emphasized': should_highlight_last_counter,
+            })}
+        >
+            {value}
+            {has_progress_dots && (
+                <div className='accumulators-stats__progress-dots'>
+                    {[1, 2, 3].map(dot => {
+                        return <span key={dot} className={`dot-${dot}`} />;
```

Comment:
```suggestion
                        return <span key={`ticks-dot-counter${dot}`} className={`dot-${dot}`} />;
```

Reply:
@yashim-deriv  could you please explain why we need to make this change if every `dot` value it unique already :) ? every mapped `dot` is a number of either 1, 2 or 3 - from the array [1, 2, 3]